  INFO root [2018-05-01 12:12:16,684] - 
  INFO root [2018-05-01 12:12:16,713] - -----------------------------------------------------------------------------------------


  INFO root [2018-05-01 12:12:17,593] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [2018-05-01 12:12:17,594] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [2018-05-01 12:12:17,594] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [2018-05-01 12:12:18,519] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:20013 )
  INFO root [2018-05-01 12:12:18,647] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.106:20013, SOMAXCONN=128.
  INFO root [2018-05-01 12:12:18,653] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:50726 )
  INFO root [2018-05-01 12:12:18,653] - NetworkInterface::initialize(INTERNAL): address 192.168.0.106:50726, SOMAXCONN=128.
  INFO root [2018-05-01 12:12:18,654] - server-configs:
  INFO root [2018-05-01 12:12:18,654] - 	internalAddr : 192.168.0.106:50726
  INFO root [2018-05-01 12:12:18,655] - 	externalAddr : 192.168.0.106:20013
  INFO root [2018-05-01 12:12:18,655] - 	componentID : 9000
  INFO root [2018-05-01 12:12:19,399] - Components::findLogger: found logger, addr:192.168.0.106:50721
  INFO root [2018-05-01 12:12:19,399] - Components::addComponent: reset orderLog, uid=17503!
  INFO root [2018-05-01 12:12:19,399] - Components::addComponent[logger], uid=17503, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-01 12:12:19,501] - ---- loginapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: B20F257E332779D9E1EA7D34638E6146. Config: Debug X86. Built: 15:53:07 Apr 29 2018. AppID: 9000. UID: 17503. PID: 12740 ----
  INFO root [2018-05-01 12:12:19,503] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [2018-05-01 12:12:19,504] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-05-01 12:12:19,615] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-05-01 12:12:31,530] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-01 12:12:31,576] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-05-01 12:12:31,754] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-01 12:12:32,413] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-01 12:12:32,682] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-05-01 12:12:32,696] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-01 12:12:32,764] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-01 12:12:33,244] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-01 12:12:34,856] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-05-01 12:12:34,856] - Script::install(): KBExtra is successfully!
S_INFO root [2018-05-01 12:12:35,062] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-05-01 12:12:35,063] - TelnetServer server is running on port 31000
  WARN root [2018-05-01 12:12:35,064] - e:\kbengine\kbe\src\server\loginapp\loginapp.cpp:153: Profile scriptCall took 0.11 seconds
  INFO root [2018-05-01 12:12:35,560] - ---- loginapp is running ----
 DEBUG root [2018-05-01 12:12:35,561] - Components::process(): Request for the process of identity...
  WARN root [2018-05-01 12:12:42,016] - Forwarding logs to logger[192.168.0.106:50721/0/0/0]...
 ERROR root [2018-05-01 12:57:57,019] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-05-01 12:57:57,028] - The following logs sent to logger failed:
  WARN root [2018-05-01 12:57:57,028] - ==> [2018-05-01 12:57:57 017] TCPPacketReceiver::processPendingEvents(192.168.0.106:50721): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-01 12:57:57,028] - ==> [2018-05-01 12:57:57 019] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2018-05-01 12:57:57,077] - TCPPacketReceiver::processPendingEvents(192.168.0.106:50723): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-01 12:57:57,082] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2018-05-01 12:57:57,087] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2018-05-01 12:57:57,088] - onLoginAppShutDown()
  INFO root [2018-05-01 12:57:57,088] - Shutdowner::onShutdown: shutting down
  INFO root [2018-05-01 12:57:57,088] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2018-05-01 12:57:57,088] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-01 12:57:57,089] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2018-05-01 12:57:57,089] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-01 12:57:57,089] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-01 12:57:57,089] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2018-05-01 12:57:57,089] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-01 12:57:57,089] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-01 12:57:57,089] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-01 12:57:57,128] - Script::uninstall(): is successfully!
 DEBUG root [2018-05-01 12:57:57,128] - ThreadPool::destroy(): starting size 2.
  INFO root [2018-05-01 12:57:58,302] - 
  INFO root [2018-05-01 12:57:58,304] - -----------------------------------------------------------------------------------------


  INFO root [2018-05-01 12:57:58,305] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [2018-05-01 12:57:58,305] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [2018-05-01 12:57:58,305] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [2018-05-01 12:57:58,313] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:20013 )
  INFO root [2018-05-01 12:57:58,313] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.106:20013, SOMAXCONN=128.
  INFO root [2018-05-01 12:57:58,315] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:51113 )
  INFO root [2018-05-01 12:57:58,315] - NetworkInterface::initialize(INTERNAL): address 192.168.0.106:51113, SOMAXCONN=128.
  INFO root [2018-05-01 12:57:58,315] - server-configs:
  INFO root [2018-05-01 12:57:58,315] - 	internalAddr : 192.168.0.106:51113
  INFO root [2018-05-01 12:57:58,315] - 	externalAddr : 192.168.0.106:20013
  INFO root [2018-05-01 12:57:58,315] - 	componentID : 9000
  WARN root [2018-05-01 12:57:58,868] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20975, errorno=10048
  INFO root [2018-05-01 12:57:58,874] - ---- loginapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: B20F257E332779D9E1EA7D34638E6146. Config: Debug X86. Built: 15:53:07 Apr 29 2018. AppID: 9000. UID: 26543. PID: 12932 ----
  INFO root [2018-05-01 12:57:58,874] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [2018-05-01 12:57:58,874] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-05-01 12:57:58,874] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-05-01 12:57:59,002] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-01 12:57:59,003] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-05-01 12:57:59,004] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-01 12:57:59,007] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-01 12:57:59,008] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-05-01 12:57:59,008] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-01 12:57:59,008] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-01 12:57:59,037] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-01 12:57:59,068] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-05-01 12:57:59,069] - Script::install(): KBExtra is successfully!
S_INFO root [2018-05-01 12:57:59,071] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-05-01 12:57:59,071] - TelnetServer server is running on port 31000
  INFO root [2018-05-01 12:57:59,359] - ---- loginapp is running ----
 DEBUG root [2018-05-01 12:57:59,359] - Components::process(): Request for the process of identity...
  INFO root [2018-05-01 12:58:06,431] - Components::findComponents: find logger(1)...
  WARN root [2018-05-01 12:58:06,742] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20035, errorno=10048
  INFO root [2018-05-01 12:58:07,752] - Components::findComponents: find logger(2)...
  INFO root [2018-05-01 12:58:07,823] - Components::findComponents: found logger, addr:192.168.0.106:51109
  INFO root [2018-05-01 12:58:07,824] - Components::addComponent: reset orderLog, uid=26543!
  INFO root [2018-05-01 12:58:07,824] - Components::addComponent[logger], uid=26543, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-01 12:58:07,825] - Components::findComponents: find dbmgr(3)...
  INFO root [2018-05-01 12:58:07,928] - Components::findComponents: found dbmgr, addr:192.168.0.106:51115
  INFO root [2018-05-01 12:58:07,928] - Components::addComponent: reset orderLog, uid=26543!
  INFO root [2018-05-01 12:58:07,929] - Components::addComponent[dbmgr], uid=26543, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-01 12:58:07,929] - Components::findComponents: found dbmgr, addr:192.168.0.106:51115
  WARN root [2018-05-01 12:58:08,929] - Forwarding logs to logger[192.168.0.106:51109/0/0/0]...
